front/src/reducers/osrdsimulation/index.ts

```diff
@@ -52,6 +53,7 @@ export const initialState: OsrdSimulationState = {
     present: { trains: [] },
     future: [],
   },
+  selectedProjectionV2: undefined,
```

Comment:
Do you think we can rename this variable something like "trainIdUsedForProjection" ?
```suggestion
  trainIdUsedForProjection: undefined,
```

Reply:
Totally agreed